app/assets/stylesheets/application.css.scss

```diff
@@ -28,3 +28,21 @@
 .downvote {
   color: red;
 }
+
+#foto,
+#foto img {
+  background: #1abc9c;
+  border-radius:50%;
+}
+
+#foto {
+  border: 15px solid #34495e;
```

Comment:
Color literals like `#34495e` should only be used in variable declarations; they should be referred to via variable everywhere else.